src/MizarWidgets/MizarMainWindow.cpp

```diff
@@ -15,10 +15,13 @@
 namespace orbit_mizar_widgets {
 
 MizarMainWindow::MizarMainWindow(
-    const orbit_mizar_data::BaselineAndComparison* baseline_and_comparison, QWidget* parent)
+    const orbit_mizar_data::BaselineAndComparison* baseline_and_comparison,
+    const Baseline<QString>& baseline_file_name, const Comparison<QString>& comparison_file_name,
```

Comment:
Again, I don't understand why you need to differentiate using types here, but I don't have enough context to have an opinion.

Reply:
I do it so the `comparison_file_name` doesn't accidentally get shown in the widget on the left (where `baseline_file_name`) should be shown.